install_config/registry/deploy_registry_existing_clusters.adoc

```diff
@@ -92,11 +93,23 @@ specified pod. When the nodes are removed, the pods are garbage collected.
 For more information on DaemonSets, see the
 link:http://kubernetes.io/docs/admin/daemons/[DaemonSet documentation].
 
-[NOTE]
+[IMPORTANT]
 ====
-Using DaemonSets with {product-title} is a work in progress, and the only
-supported process is creating a registry. Currently, using deployment
-configurations to deploy a registry is recommended.
+Using daemonsets is a Technology Preview feature only. The only supported
```

Comment:
I think I might change the wording here since we "support" installing the registry with a DS only by providing a command.  It is still not recommended and still not a truly supported feature since it is tech preview.  Just the blurb about what tech preview means is probably good enough for both sections